WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -926,6 +926,9 @@
 		638E2F3D4DFE9E93AF660CED /* Pods_WordPressAllTimeWidget.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 0075A07B739C4EDB5EC613B8 /* Pods_WordPressAllTimeWidget.framework */; };
 		660A036E6EE9D353F2712081 /* Pods_WordPressNotificationServiceExtension.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = E2A297CCB53FCF6851D79331 /* Pods_WordPressNotificationServiceExtension.framework */; };
 		6867C0633E597372235CB5E0 /* Pods_WordPressDraftActionExtension.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F7E3CC306AECBBCB71D2E19C /* Pods_WordPressDraftActionExtension.framework */; };
+		6EC71EC32689A67400ACC0A0 /* SupportScreen.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6EC71EC22689A67400ACC0A0 /* SupportScreen.swift */; };
+		6EC71F8C2689A7A900ACC0A0 /* SupportScreen.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6EC71EC22689A67400ACC0A0 /* SupportScreen.swift */; };
+		6EC71FA02689A7AD00ACC0A0 /* SupportScreen.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6EC71EC22689A67400ACC0A0 /* SupportScreen.swift */; };
```

Comment:
This apparent duplication (triplication) is because the screen object is, _correctly_, part of the UI tests target and the two screenshot generation ones.

![image](https://user-images.githubusercontent.com/1218433/125240380-6146de80-e32d-11eb-81a1-ff51a6deef8a.png)

FYI: The next step for the UITestsFoundation setup is to move all the shared screen objects in there. That will make it easier to share screen objects across targets, because once a test target adds UITestsFoundation as a library dependency (which all three already do) it automatically gets access to the screen objects in the library. _Stay tuned_ 🤓

Reply:
Gotcha, that makes sense!
Oooh, that should be exciting!